views/media.twig

```diff
@@ -0,0 +1,166 @@
+{% extends 'layout.twig' %}
```

Comment:
PLEASE `pull origin _dev` before pushing your changes (`views/media.twig` moved to `en/views/media.twig`)